src/express/api.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const axios = require(`axios`);
+
+const TIMEOUT = 1000;
+
+const port = process.env.API_PORT || 3000;
+const defaultUrl = `http://localhost:${port}/api/`;
+
+class API {
+
+  constructor(baseURL, timeout) {
+    this._http = axios.create({
+      baseURL,
+      timeout
+    });
+  }
+
+  async _load(url, options) {
+    const response = await this._http.request({url, ...options});
+    return response.data;
+  }
+
+  getArticles() {
+    return this._load(`/articles`);
+  }
+
+  getArticle(id) {
+    return this._load(`/articles/${id}`);
+  }
+
+  search(query) {
+    return this._load(`/search`, {params: {query}});
+  }
+
+  // TODO а почему эти методы async а другие нет?
+  async getCategories() {
+    return this._load(`/category`);
+  }
```

Comment:
```suggestion
 getCategories() {
    return this._load(`/category`);
  }
  
   async getCategories() {
    const categories = await this._load(`/category`);
    return categories;
  }
```
Это две почти (99.10% разницы не заметишь) одинаковые записи